### PR TITLE
EPMRPP-33317: Subtypes chart + Add subtype button

### PR DESCRIPTION
**Pull request by margalencia**
image

### CODE REVIEW

File: app/src/pages/common/settingsPage/defectTypesTab/defectTypesGroup.jsx [hunk 1]
```diff
@@ -16,27 +22,88 @@ ColorMarker.propTypes = {
...
-            <span className={cx('defect-type-name')}>{longName}</span>
-          </span>
-        </div>
-        <div className={cx('abbr-cell')}>{shortName}</div>
-        <div className={cx('color-cell')}>
-          <ColorMarker color={color} />
+    const { group, intl } = this.props;
+
+    return (
+      <div>
```
Comment (DiscoElevator) [inline app/src/pages/common/settingsPage/defectTypesTab/defectTypesGroup.jsx]: Is this div necessary? You can use `Fragment` instead

File: app/src/pages/common/settingsPage/defectTypesTab/defectTypesTab.scss [hunk 2]
```diff
@@ -65,3 +74,21 @@
 .defect-type-name {
   margin-left: 5px;
 }
+
+.defect-type-chart {
+  position: absolute !important;
```
Comment (DiscoElevator) [inline app/src/pages/common/settingsPage/defectTypesTab/defectTypesTab.scss]: Why do you use `!important` here?

Comment (margalencia) [reply app/src/pages/common/settingsPage/defectTypesTab/defectTypesTab.scss]: I have to use it to overwrite the chart's style, it has `position: relative`.
I need to remove the chart from the normal flow so it doesn't impact the first row height.

File: app/src/pages/common/settingsPage/defectTypesTab/defectTypesGroup.jsx (hunk 1 above)
Comment (margalencia) [reply app/src/pages/common/settingsPage/defectTypesTab/defectTypesGroup.jsx]: yep, for now, let's live it as it is, I will fix it in the next PR, I've restructured the files, created subcomponents

**Comment by DiscoElevator** [comment]
изображение

**Comment by margalencia** [comment]
will be fixed in the next pr 
image

**Pull request merged**